### PR TITLE
Expand Example Documentation at "openapi-ts.dev/examples"

### PR DESCRIPTION
**Pull request by bennobuilder**
## Changes

The example documentation on the `openapi-typescript` homepage has been expanded to include:

1. Addition of [`feature-fetch`](https://github.com/builder-group/community/tree/develop/packages/feature-fetch) in the Data Fetching section ([Example](https://github.com/builder-group/community/tree/develop/examples/feature-fetch/vanilla/open-meteo))
2. Two new sections:
   - Hono with [`@bldc/openapi-router`](https://github.com/builder-group/community/tree/develop/packages/openapi-router/src/features/with-hono) ([Example](https://github.com/builder-group/community/tree/develop/examples/openapi-router/hono/petstore))
   - Express with [`@bldc/openapi-router`](https://github.com/builder-group/community/tree/develop/packages/openapi-router/src/features/with-express) ([Example](https://github.com/builder-group/community/tree/develop/examples/openapi-router/express/petstore))

The goal is to showcase additional examples that might bring value to the `openapi-typescript` user. Further expansion will be based on your (@drwpow) feedback and whether you think this additions can bring value to your users.

cheers

### CODE REVIEW

**Comment by bennobuilder** [comment]
Hey @kerwanp @drwpow,

Do you think this addition to the example docs would bring value to the community? If not, feel free to close this Draft.

Thanks 😊

**Review by drwpow — APPROVED** [review]

**Comment by bennobuilder** [comment]
> Ah sorry; didn’t realize this was ready for review as a draft. In the future, please just open a full PR so we know it’s ready for feedback.
> 
> This is great! I’m **so** excited to see more libraries take this approach, and thanks for bringing this library to our attention! We’ll always add other library recommendations like this provided they have an open license. More options in OSS helps everyone, and the “competition” makes the internet better 🙂.
> 
> Will mark as ready for review, and I think as-written this is good, but just let me know when you’d like to merge it and I’d be happy to.

Yeah, sorry for the confusion. The draft was intended to check if you guys are open to including libraries I created in your example section (as stated in the description). I believe they could benefit users of `openapi-typescript`, but I was unsure whether you guys had the same view.

I’ve updated the example documentation, and the PR is now ready for review.

cheers :)

As a site note, the benchmark from [here](https://openapi-ts.dev/openapi-fetch/) seems to be a bit outdated? On my machine (MacBook M3 Pro) for example `openapi-typescript-fetch` was the fastest (see image) and it also got more lightweight.

image

**Comment by drwpow** [comment]
> The draft was intended to check if you guys are open to including libraries I created in your example section (as stated in the description). I believe they could benefit users of openapi-typescript, but I was unsure whether you guys had the same view.

I’m a big believer in promoting each others’ open source work, because that’s the spirit of what we’re doing! If people take this code, and build off of it, or develop their own libraries, that’s the entire point! 😄 Everyone only benefits from more libraries. And if they’re philosophically aligned like your library is to this project, even better 😎 

> As a site note, the benchmark from [here](https://openapi-ts.dev/openapi-fetch/) seems to be a bit outdated?

It’s not outdated per se; just flaky. We are working on improving it here: https://github.com/openapi-ts/openapi-typescript/issues/1818. Thoughts welcome, but I am now suspicious of msw (unconfirmed) playing a part in the flakiness. We’ll improve the benchmarks across the board.

Oh but you’re right—openapi-typescript-fetch did get a recent update that sped it up by quite a bit. Didn’t mean to deny them of that; we should update the benchmarks we have 😅. But I also would like to spend more time on #1818 so we get more reliable numbers overall.

File: docs/examples.md [hunk 1]
```diff
@@ -11,8 +11,135 @@ The types generated by openapi-typescript are universal, and can be used in a va
...
+<details>
+<summary><a href="/openapi-fetch/">openapi-fetch</a> (recommended)</summary>
+
+::: code-group
+
+```ts [test/my-project.ts]
+import createClient from "openapi-fetch";
+import type { paths } from "./my-openapi-3-schema"; // generated by openapi-typescript
+
+const client = createClient<paths>({ baseUrl: "https://myapi.dev/v1/" });
```
Comment (drwpow) [inline docs/examples.md:23]: ❤️ These little improvements go a long way, thank you!

File: docs/openapi-fetch/index.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ openapi-fetch is a type-safe fetch client that pulls in your OpenAPI schema. Wei
 | :------------------------- | ---------: | :------------------------- |
 | openapi-fetch              |     `5 kB` | `300k` ops/s (fastest)     |
 | openapi-typescript-fetch   |     `4 kB` | `150k` ops/s (2× slower)   |
+| feature-fetch              |    `15 kB` | `300k` ops/s (not slower)  |
```
Comment (drwpow) [inline docs/openapi-fetch/index.md:13]: 😄 🚀 

You could also put `(fastest)` here as well if you wanted to.

**Review by drwpow — APPROVED** [review]
Looks great! Thanks for the improvements to the docs as well.

Will merge this, but feel free to add any followups if needed

File: docs/openapi-fetch/index.md (hunk 2 above)
Comment (bennobuilder) [reply docs/openapi-fetch/index.md:13]: The performance difference between `openapi-fetch` and `feature-fetch` was negligible (0.0x) in my tests. I didn't update the statistics based on a single result set. Your updated benchmark in #1818 should provide more comprehensive data on relative performance 🚀

File: docs/examples.md (hunk 1 above)
Comment (bennobuilder) [reply docs/examples.md:23]: Glad you appreciate the changes. Since its the "Example" page I thought I add some to the "Data fetching" section :)

**Pull request merged**